cli/src/commands/debug/mod.rs

```diff
@@ -74,6 +77,7 @@ pub enum DebugCommand {
     Revset(DebugRevsetArgs),
     Snapshot(DebugSnapshotArgs),
     Template(DebugTemplateArgs),
+    ToyBackendInit(DebugToyBackendInitArgs),
```

Comment:
How about calling it `jj debug init-local`? Just being under `debug` seems sufficient to make the user understand that they shouldn't use it. The whole `debug` group is hidden so most users probably don't even know that it exists.

Reply:
I agree that the name doesn't matter much. I can do `init-local` or whatever people like.

TBH, I don't understand the opposition to `toy-backend`; to me it describes the purpose of the backend well. The only other name that came to me was `demo-backend`.

Whatever we go with, it's easy to rename later if the role of this backend changes.

Reply:
I'll go with `init-local` unless I hear otherwise.

Reply:
FWIW, I think of it more as proof of concept that might one day evolve into a default native backend. There are many steps required before we get there, though, so it's less important what we call it right now. `init-local` matches the `LocalBackend` name well at least. But I'm also fine with calling it "toy" if that's what people prefer.

Reply:
In my mind `toy-backend` is a shorter/clearer version of `poc-backend` or `proof-of-concept-backend`.